Hands-on lab/HOL step-by-step - Cloud-native applications.md

```diff
@@ -44,13 +44,8 @@ Microsoft and the trademarks listed at https://www.microsoft.com/en-us/legal/int
     - [Task 1: Increase service instances from the Azure Portal](#task-1-increase-service-instances-from-the-azure-portal)
     - [Task 2: Resolve failed provisioning of replicas](#task-2-resolve-failed-provisioning-of-replicas)
     - [Task 3: Restart containers and test HA](#task-3-restart-containers-and-test-ha)
-    - [Task 4: Configure Cosmos DB Autoscale](#task-4-configure-cosmos-db-autoscale)
-    - [Task 5: Test Cosmos DB Autoscale](#task-5-test-cosmos-db-autoscale)
   - [Exercise 4: Working with services and routing application traffic](#exercise-4-working-with-services-and-routing-application-traffic)
-    - [Task 1: Update an external service to support dynamic discovery with a load balancer](#task-1-update-an-external-service-to-support-dynamic-discovery-with-a-load-balancer)
-    - [Task 2: Adjust CPU constraints to improve scale](#task-2-adjust-cpu-constraints-to-improve-scale)
     - [Task 3: Perform a rolling update](#task-3-perform-a-rolling-update)
-    - [Task 4: Configure Kubernetes Ingress](#task-4-configure-kubernetes-ingress)
```

Comment:
not possible without acme setup first

Reply:
maybe replace with [http application routing](https://docs.microsoft.com/en-us/azure/aks/http-application-routing) or just `k port-forward`

Reply:
add parts with service with node ip again - this is the way they route into cluster